modules/types/Video.js

```diff
@@ -148,5 +148,10 @@ class Video {
             }
         }
     }
+
+    GetURLID()
+    {
+        return this.url.replace("https://www.youtube.com/watch?v=", "");
+    }   
```

Comment:
this does not work, yt-dlp supports all kinds of urls

Reply:
(method can be removed if we use default logic)

Reply:
It worked for me, i also tested with several videos and yt-dlp gave the url only in https ones.

If you say it will give different ones, i will make a bit dynamic one.